controllers/cartController.js

```diff
@@ -0,0 +1,17 @@
+const cartService = require("../services/cartService");
+
+const selectAllCart = async (req, res) => {
```

Comment:
```suggestion
const getCarts = async (req, res) => {
```
네이밍이 너무 복잡합니다. 그냥 데이터를 가져오는 것이면 get으로 충분할 것 같습니다.
이에 맞게 전부 수정해 주세요.